analyticsdata/src/run_report_with_aggregations.php

```diff
@@ -0,0 +1,109 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+* Google Analytics Data API sample application demonstrating the usage of
+* metric aggregations in a report.
+* See https://developers.google.com/analytics/devguides/reporting/data/v1/rest/v1beta/properties/runReport#body.request_body.FIELDS.metric_aggregations
+* for more information.
+* Usage:
+*   composer update
+*   php run_report_with_aggregations.php YOUR-GA4-PROPERTY-ID
+*/
+
+namespace Google\Cloud\Samples\Analytics\Data;
+
+// [START analyticsdata_run_report_with_aggregations]
+use Google\Analytics\Data\V1beta\BetaAnalyticsDataClient;
+use Google\Analytics\Data\V1beta\DateRange;
+use Google\Analytics\Data\V1beta\Dimension;
+use Google\Analytics\Data\V1beta\Metric;
+use Google\Analytics\Data\V1beta\MetricType;
+use Google\Analytics\Data\V1beta\MetricAggregation;
+use Google\Analytics\Data\V1beta\RunReportResponse;
+
+/**
+* @param string $propertyID Your GA-4 Property ID
+* Runs a report which includes total, maximum and minimum values
+* for each metric.
+*/
+function run_report_with_aggregations(string $propertyId)
+{
+    // [START analyticsdata_initialize]
+    // Imports the Google Analytics Data API client library.
+    $client = new BetaAnalyticsDataClient();
+    // [END analyticsdata_initialize]
+
+    // Make an API call.
+    $response = $client->runReport([
+        'property' => 'properties/' . $propertyId,
+        'dimensions' => [new Dimension(
+            [
+                'name' => 'country',
+            ]
+        )],
+        'metrics' => [new Metric(
+            [
+                'name' => 'sessions',
+            ]
+        )],
+        'dateRanges' => [
+            new DateRange([
+                'start_date' => '365daysAgo',
+                'end_date' => 'today',
+            ]),
+        ],
+        'metricAggregations' => [
+    MetricAggregation::TOTAL,
+    MetricAggregation::MAXIMUM,
+    MetricAggregation::MINIMUM
+        ]
+    ]);
+
+    printRunReportResponseWithAggregations($response);
+}
+
+/**
+ * Print results of a runReport call.
+ * @param RunReportResponse $response
+ */
+function printRunReportResponseWithAggregations($response)
+{
+    // [START analyticsdata_print_run_report_response_header]
+    printf('%s rows received%s', $response->getRowCount(), PHP_EOL);
+    foreach ($response->getDimensionHeaders() as $dimensionHeader) {
+        printf('Dimension header name: %s%s', $dimensionHeader->getName(), PHP_EOL);
+    }
+    foreach ($response->getMetricHeaders() as $metricHeader) {
+        printf('Metric header name: %s (%s)%s', $metricHeader->getName(), MetricType::name($metricHeader->getType()), PHP_EOL);
```

Comment:
more readable format:
```suggestion
        printf(
            'Metric header name: %s (%s)' . PHP_EOL,
            $metricHeader->getName(),
            MetricType::name($metricHeader->getType())
        );
```